vibrato/src/dictionary/connector.rs

```diff
@@ -98,30 +100,116 @@ impl ConnectorCost for MatrixConnector {
     }
 }
 
+#[derive(Decode, Encode)]
+pub struct RawConnector {
```

Comment:
I do not think it is appropriate to use "Raw" in this structure name. I guess the intent is to build from raw text data, but that is not a feature of this structure.

Since MatrixConnector uses a matrix for the data structure, how about TrieConnector?
(However, since I do not yet fully understand this data structure, I would like you to name it appropriately)

Reply:
I don't think TrieConnector describes this Connector, but I can't think of a good name.
For now, I will keep the name RawConnector.

Reply:
Okay. Devising it is the future work.